<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configurationPage</class>
 <widget class="QWidget" name="configurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
                    background-color:rgba(229, 165, 10, 0.95);
                    border: 1px solid rgba(0, 0, 0, 0.8);
                    border-radius: 10px;
                    padding: 10px;
                    }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="radio1Player">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton {
                                            color: rgb(165, 29, 45);
                                            padding: 10px;
                                            margin: 5px;
                                            }

                                            QRadioButton::indicator {
                                            width: 20px;
                                            height: 20px;
                                            }

                                            QRadioButton::indicator:checked {
                                            background-color: white;
                                            border: 2px solid green;
                                            border-radius: 10px;
                                            }


                                        </string>
         </property>
         <property name="text">
          <string>1 Player</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio2Player">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton {
                                            color: rgb(165, 29, 45);
                                            padding: 10px;
                                            margin: 5px;
                                            }

                                            QRadioButton::indicator {
                                            width: 20px;
                                            height: 20px;
                                            }

                                            QRadioButton::indicator:checked {
                                            background-color: white;
                                            border: 2px solid green;
                                            border-radius: 10px;
                                            }


                                        </string>
         </property>
         <property name="text">
          <string>2 Players</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnCrear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">    color: white;
                                            background-color: rgb(165, 29, 45);
                                            padding: 10px;
                                            border-radius: 10px;
                                            margin: 5px;</string>
         </property>
         <property name="text">
          <string>CREATE MATCH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnirse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">    color: white;
                                            background-color: rgb(165, 29, 45);
                                            padding: 10px;
                                            border-radius: 10px;
                                            margin: 5px;</string>
         </property>
         <property name="text">
          <string>JOIN A MATCH</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btnVolver">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">    color: white;
                                            background-color: rgb(165, 29, 45);
                                            padding: 10px;
                                            border-radius: 10px;
                                            margin: 5px;</string>
         </property>
         <property name="text">
          <string>BACK TO LOBBY</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
